hermes-core/src/main/java/com/ctrip/hermes/core/meta/remote/RemoteMetaProxy.java

```
@@ -264,4 +264,32 @@ public String apply(String ip) {
 		}
 	}
 
+	@Override
+	public int registerSchema(String schema, String subject) {
+		Map<String, String> params = new HashMap<>();
+		params.put("schema", schema);
+		params.put("subject", subject);
+		String response = post("/schema/register", null, params);
```

maybe we need update the post/get method, currently, we ignore all exceptions.